File: DayOne/JavaScript/number-guessing-app.js

    let prompt = require("prompt-sync")();

    let random = Math.floor(Math.random()* 100) + 1

    let count = 0;
    
        for (let attempts = 0; attempts < 5; attempts++){

            let userInput = prompt("Enter a number: ");
    

            if (1 < userInput && userInput > 100)

            console.log("Doesn't count as an attempt\nTry again!'");

           else if (userInput == random){
        
           console.log("Correct\nYou're a genius!!!");

           count+=1;

           break;
           }
    
           else if  (userInput < random){

           console.log("Higher");
           count+=1;
           }
           else{ 

           console.log("lower");
           count+=1;
           }

        }

let rating = " ";
if (count == 1)
    rating = "Legendary";

else if (count == 2)
    rating = "Excellent";

else if (count == 3 || count == 4)
    rating = "Good";

else
    rating = "Close!\nBetter luck next time";


console.log(`\tFinal Summary\nThe Correct Number is ${random} attempts used ${count} \n${rating}`);

console.log()
    

        

File: DayOne/Python/number_guessing_app.py
import random

number = random.randint(1,100)

correct_number = number

count = 0

for attempts in range(5):

    user_input = int(input("Enter a number: "))

#    if user_input := str:
#
#        print("Error")

    if 1 < user_input > 100:

        print("Doesn't count as an attempt\nTry again!'")

    elif user_input == correct_number:
        
        print("Correct\nYou're a genius!!!")

        count+=1

        break;

    elif user_input < correct_number:

        print("Higher")
        count+=1

    else:

        print("lower")
        count+=1

rating = " "
if count == 1:
    rating = "Legendary"

elif count == 2:
    rating = "Excellent"

elif count == 3 or count == 4:
    rating = "Good"

else:
    rating = "Close!\nBetter luck next time"


print(f"\tFinal Summary\nThe Correct Number is {correct_number} attempts used {count}\n{rating} ")



    
    
    


